commands/proxy/proxy-set.ts

```diff
@@ -27,9 +27,9 @@ export class ProxySetCommand extends ProxyCommandBase {
 	}
 
 	public async execute(args: string[]): Promise<void> {
-		let urlString = args[0],
-			username = args[1],
-			password = args[2];
+		let urlString = args[0];
```

Comment:
const?

Reply:
nope, it is set on line 39
Also now linter checks if any `const` can be used instead of `let` and shows error in this case.